packages/starksheet-webapp/src/provider/EVMProvider.ts

```diff
@@ -91,6 +92,19 @@ export class EVMProvider implements ChainProvider {
     return `${this.config.nftBaseUrl}${contractAddress}`;
   }
 
+  /**
+   * @inheritDoc
+   */
+  getSpreadsheetContract(): EvmSpreadsheetContract {
+    const address = process.env.REACT_APP_SPREADSHEET_ADDRESS;
+
+    if (!address) {
+      throw new Error("missing env variable REACT_APP_SPREADSHEET_ADDRESS");
+    }
```

Comment:
I am always worried to have this kind of things spread everywhere in the app; I prefer to keep all the ENV parsing in one single file so that it's clear where the env variables plays a role. And then import the parsed values from there

Reply:
I get the point. Note that this is specific to EVM because starknet pulls this info from the contract.json artifact. Once this part is refactored in a cross-chain way, this could be further cleaned up by having a central place for defining the chain config